Xamarin.PropertyEditing.Mac/Controls/BaseEditorControl.cs

```diff
@@ -19,7 +21,25 @@ public NSButton ActionButton
 			get { return actionButton; }
 		}
 
-		public event EventHandler PropertyButtonClicked;
+		event EventHandler propertyButtonClicked;
+		public event EventHandler PropertyButtonClicked {
+			add {
+				lock (propertyButtonClicked) {
+					propertyButtonClicked += value;
+					propertyButton.Hidden = false;
```

Comment:
This doesn't seem related to accessibility changes. We won't be handling whether property buttons are visible or not this way, currently we should assume always visible.

Reply:
Fair enough, the reason I did it was buttons that are visible and don't do anything is fairly bad for accessibility, but I can remove it

Reply:
@iainx My CustomExpression PR refactors the PropertyButton functionality into its own class. I also agree that Buttons that do nothing, in the case of ReadOnly properties, shouldn't be visible. Showing something that isn't interactive is both bad accessibility and a bad UX.